test/ffc-rules-engine.test.js

```diff
@@ -244,179 +271,159 @@ describe('Rule: Claimed perimeter <= perimeter adjusted to take into account per
     expect(result.events[0].type).toBe('cultivated')
   })
   test('cultivatedParcel fails when parcel is not cultivated', async () => {
-    const parcel = {
-      parcelRef: 'PR123',
-      perimeter: 75,
-      perimeterFeatures: [
-        {
-          type: 'lake',
-          perimeter: 15
-        }
-      ],
-      landCoverClass: 100,
-      previousActions: [],
-      sssi: false
-    }
+    const randomNonCulivatedClass = 100
+    const parcel = getParcelWithDefaults({ landCoverClass: randomNonCulivatedClass })
     const result = await runEngine(
       [rules.cultivatedParcel],
       { parcel }
     )
 
     expect(result.events.length).toBe(0)
   })
-  test('Passes when claimed perimeter is less than adjusted perimeter', async () => {
-    const parcel = {
-      parcelRef: 'PR123',
-      perimeter: 75,
-      perimeterFeatures: [
-        {
-          type: 'lake',
-          perimeter: 15
-        }
-      ],
-      previousActions: [],
-      sssi: false
-    }
+})
+
+describe('Rule: Claimed area <= actual area', () => {
+  const parcel = getParcelWithDefaults({})
```

Comment:
I think we should specify the parcel area here to ensure that the area tests are valid regardless of the default data.